### PR TITLE
[W4.5c][W15-A2]Ho Bing Xuan

### PR DESCRIPTION
**Pull request by IzHoBX**
Added exception handling for file being changed to read-only during runtime.

### CODE REVIEW

File: src/seedu/addressbook/Main.java [hunk 1]
```diff
@@ -32,7 +32,7 @@
     private StorageFile storage;
     private AddressBook addressBook;
 
-    /** The list of person shown to the user most recently.  */
+    /** The list of person shown to the user most recently.     */
```
Comment (IzHoBX) [inline src/seedu/addressbook/Main.java:35]: This should be changed back to 1 space only.

File: src/seedu/addressbook/Main.java [hunk 2]
```diff
@@ -32,7 +32,7 @@
     private StorageFile storage;
     private AddressBook addressBook;
 
-    /** The list of person shown to the user most recently.  */
```
Comment (IzHoBX) [inline src/seedu/addressbook/Main.java:32]: This is better

**Review by IzHoBX — COMMENTED** [review]
Please change back to original 1 space version

**Review by devamanyu — COMMENTED** [review]
Expected LO has been achieved. Please close this PR.

**Pull request closed without merging**